app/controllers/search_controller.rb

```diff
@@ -0,0 +1,3 @@
+class SearchController < ApplicationController
+  def index; end
```

Comment:
Tiny styling question: are we going with this syntax on the BE too or vice versa?

Reply:
I think we should - this is a Rubocop suggestion